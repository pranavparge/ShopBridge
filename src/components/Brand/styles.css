.banner {
  text-align: center;
  vertical-align: middle;
}

.banner h1 {
  margin-top: 20vh;
  font-size: 6em;
}

.brand {
  background-color: #0d6efd;
  color: #fff;
}
