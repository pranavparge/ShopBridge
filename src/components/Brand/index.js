import React from "react";
import "./styles.css";

function Brand() {
  return (
    <div className="banner">
      <h1>
        Welcome to <span className="brand">ShopBridge</span>
      </h1>
    </div>
  );
}

export default Brand;
