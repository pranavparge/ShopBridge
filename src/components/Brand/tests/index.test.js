import { render } from "@testing-library/react";
import Brand from "../index";

test("to check if it matches the snapshot", () => {
  const { asFragment } = render(<Brand />);
  expect(asFragment(<Brand />)).toMatchSnapshot();
});
