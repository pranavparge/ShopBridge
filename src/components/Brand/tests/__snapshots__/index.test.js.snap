// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`to check if it matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="banner"
  >
    <h1>
      Welcome to 
      <span
        class="brand"
      >
        ShopBridge
      </span>
    </h1>
  </div>
</DocumentFragment>
`;
